app/views/users/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Show details of a particular user</h1>
+<div class="user-page">
+  <%= render "shared/details", user: @user %>
+    <div class="user-bio">
+     <strong class="bio">Bio: </strong> <br>
+     <p><%= @user.bio %></p>
+    </div>
+    <div class="user-posts">
+    <% @user.recent_posts.each do |post| %>
+        <%= link_to user_post_path(user_id:@user.id, id: post.id) do%>
+      <%= render "shared/post_details", post: post %>
+        <%end%>
+    <%end%>
+    </div>
+    <button><%= link_to 'See all posts', user_posts_path(@user)%></button>
```

Comment:
- [ ] When clicking on the `See all posts` button an error is produced, kindly make sure to fix this so that when the user navigates the posts index page all the posts are shown

![Screenshot_20230309_144353](https://user-images.githubusercontent.com/25227022/224042027-8c5e727b-5a6b-4fbe-801a-7b5820ff11a9.png)